### ISSUE TITLE
[Feature Request]: Remove pings on entity despawn/death

### ISSUE THREAD

**Issue by nco2k** [LukenSkyne/Minecraft-Ping-Wheel#42]
the pinged item gets picked up or despawns, or the pinged entity gets killed or despwans.

**Comment by LukenSkyne** [comment]
Yeah I've had it like that before, but I didn't like the behavior. I also thought about showing something like a questionmark on the last known location.

I might make this configurable.

**Comment by nco2k** [comment]
> I might make this configurable.

yes pls.

**Comment by Jetpack-Cat** [comment]
I was just coming to report this. If the mob dies, have the ping disappear. If the item gets picked up or despawns maybe change the color of the ping showing it's no longer there, or like you said above a questions mark. The mob dying or despawning though should just remove the ping, but configuration to choose is always the best way to go. If you could, if a pinged item is no longer there you could add a configurable time for it to disappear. So instead of disappearing immediately the author could choose for it to disappear 5 seconds after the item or mob that's pinged is no longer alive/there. That would actually give some context to other players as to why the ping is no longer there, they see it change to a question mark then 5 seconds later it disappears.
